src/ape/managers/chain.py

```diff
@@ -1014,6 +1014,7 @@ def get_contract_type(addr: AddressType):
 
         return contract_types
 
+    @nonreentrant(lambda *args, **kwargs: args[1])
```

Comment:
This might make it a little clearer what this random lambda is for
```suggestion
    @nonreentrant(key_fn=lambda *args, **kwargs: args[1])
```